proto/hummock.proto

```diff
@@ -686,15 +685,6 @@ message HummockVersionStats {
   map<uint32, TableStats> table_stats = 2;
 }
 
-message GetScaleCompactorRequest {}
-
-message GetScaleCompactorResponse {
```

Comment:
Will we add this back in the future for compactor scaling?

Reply:
According to the test, our policy will be on the cloud side, so there is no plan to add back the meta interface for now. Even adding a new interface will not affect the compatibility, so I remove it in this pr